tests/spread/integration/file/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Integration tests for file
+
+execute: |
+  rootfs="$(install-slices file_bins)"
+
+  chroot "${rootfs}/" file --help
+  echo "Hello World" > testfile && \
+    chroot "${rootfs}/" file testfile
```

Comment:
I would suggest using `file -b testfile` and then checking that it reports a value of `ASCII text`.
For example:
```shell
$ echo "Hello World" > testfile && file -b testfile
ASCII text
```

Also, don't forget that when you create test files that the chisel'd utility needs to operate on, you need to create the file inside the `${rootfs}` chroot directory, since `chroot` will change the root directory.
I.e., you need something like this:
```
  echo "Hello World" > "${rootfs}/testfile" && \
    chroot "${rootfs}/" file testfile
```